backend/laundry/api_wrapper.py

```diff
@@ -1,46 +1,58 @@
 import requests
-from bs4 import BeautifulSoup
 from django.conf import settings
 from django.utils import timezone
 from requests.exceptions import ConnectTimeout, HTTPError, ReadTimeout
 
 from laundry.models import LaundryRoom, LaundrySnapshot
 
 
-HALL_URL = f"{settings.LAUNDRY_URL}/?location="
+def get_room_url(room_id: int):
+    return f"{settings.LAUNDRY_URL}/rooms/{room_id}/machines?raw=true"
 
 
-def update_machine_object(cols, machine_object):
+headers = {
```

Comment:
make this a constant `LAUNDRY_HEADERS` or something, you can then import this in your command as well